lib/webpacker/dev_server.rb

```diff
@@ -0,0 +1,73 @@
+# Same convention as manifest/configuration.rb
+
+# Loads webpacker configuration from config/webpacker.yml
+
+require "webpacker/configuration"
+
+class Webpacker::DevServer < Webpacker::FileLoader
+  class << self
+    def enabled?
+      case ENV["WEBPACKER_DEV_SERVER"]
```

Comment:
OK with any way to solve the problem of having some developers on my team have to keep a modified version of the yml file.

1. The ENV value is totally optional
2. DISABLE_XYZ == false . ==> does that mean enable XYZ?